.buildkite/hooks/pre-exit

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+[[ -n "${DEBUG:-}" ]] && set -x
+
+# The premerge step runs on linux agents!
+if [[ ${BUILDKITE_AGENT_META_DATA_OS} == "windows" ]]; then
+    powershell ci/purge.ps1
```

Comment:
If you add
* `-NoProfile` the ps will execute slightly faster
* `-NonInteractive` there will be no possibility of interactive prompts blocking

Both are good practise.